backend/app/glossary/models.py

```diff
@@ -11,31 +11,43 @@
     from app.schema import User
 
 
+class ProcessingStatuses:
+    IN_PROCESS = "IN_PROCESS"
+    DONE = "DONE"
+
+
 class GlossaryDocument(Base):
     __tablename__ = "glossary_document"
     id: Mapped[int] = mapped_column(primary_key=True)
-    created_by: Mapped[int] = mapped_column(ForeignKey("user.id"))
-    created_at: Mapped[datetime] = mapped_column(default=datetime.now(UTC))
-    updated_at: Mapped[datetime] = mapped_column(default=datetime.now(UTC))
+    created_at: Mapped[datetime] = mapped_column(default=datetime.now)
+    updated_at: Mapped[datetime] = mapped_column(default=datetime.now)
+
+    original_document: Mapped[str] = mapped_column()
```

Comment:
nit: It is not needed. If you need it for processing, make it `nullable` and remove after processing.